homeassistant/components/homeassistant/triggers/time_pattern.py

```diff
@@ -37,6 +37,7 @@ def __call__(self, value):
                 number = int(value[1:])
             else:
                 number = int(value)
+                value = int(value)
```

Comment:
```suggestion
                value = number = int(value)
```

slightly faster but probably doesn't matter here